### ISSUE TITLE
TypeError: request.resolver is not a function

### ISSUE THREAD

**Issue by ofarukcaki** [jtassin/pending-xhr-puppeteer#286]
I get these warning messages constantly on my program when I use  `await pendingXHR.waitForAllXhrFinished();`
It doesn't crash my program but console logs are filled with the same error. If this isn't a bug, how can I fix that?

**My versions:**
"pending-xhr-puppeteer": "^2.1.0",
"puppeteer": "^1.20.0",

```
(node:3760) UnhandledPromiseRejectionWarning: TypeError: request.resolver is not a function
    at Page.PendingXHR.requestFinishedListener (C:\Users\omerf\Desktop\folder\project\node_modules\pending-xhr-puppeteer\lib\index.js:86:25)
    at Page.emit (events.js:228:7)
    at Page.EventEmitter.emit (domain.js:475:20)
    at NetworkManager.<anonymous> (C:\Users\omerf\Desktop\folder\project\node_modules\puppeteer\lib\Page.js:113:76)
    at NetworkManager.emit (events.js:223:5)
    at NetworkManager.EventEmitter.emit (domain.js:475:20)
    at NetworkManager._onLoadingFinished (C:\Users\omerf\Desktop\folder\project\node_modules\puppeteer\lib\NetworkManager.js:291:10)
    at CDPSession.emit (events.js:223:5)
    at CDPSession.EventEmitter.emit (domain.js:475:20)
    at CDPSession._onMessage (C:\Users\omerf\Desktop\folder\project\node_modules\puppeteer\lib\Connection.js:200:12)
    at Connection._onMessage (C:\Users\omerf\Desktop\folder\project\node_modules\puppeteer\lib\Connection.js:112:17)
    at WebSocket.<anonymous> (C:\Users\omerf\Desktop\folder\project\node_modules\puppeteer\lib\WebSocketTransport.js:44:24)
    at WebSocket.onMessage (C:\Users\omerf\Desktop\folder\project\node_modules\ws\lib\event-target.js:120:16)
    at WebSocket.emit (events.js:223:5)
    at WebSocket.EventEmitter.emit (domain.js:475:20)
    at Receiver.receiverOnMessage (C:\Users\omerf\Desktop\folder\project\node_modules\ws\lib\websocket.js:789:20)
(node:3760) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 65)
```

**Comment by phpmalik** [comment]
Encountered this after updating packages 

**My Versions:**
"pending-xhr-puppeteer": "^2.3.2",
"puppeteer": "^5.3.1",

**Comment by jtassin** [comment]
sorry for the long delay, I will have a look

**Comment by jtassin** [comment]
Thx for report is should be fixed with the new version (2.3.3)
